.github/workflows/doxygen.yml

```diff
@@ -0,0 +1,38 @@
+name: Doxygen
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+  - cron:  '0 0 * * 0'
```

Comment:
```suggestion
  schedule:
  - cron:  '0 0 * * 0'
```

Reply:
I see no difference for your suggested change